pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -79,6 +79,19 @@ tasks:
         templateCoveragePercent: percent
       python: |
         from lsst.analysis.tools.atools import DiffimMetadataMetricTool
+  associationCore:
+    class: lsst.analysis.tools.tasks.MetadataAnalysisTask
+    config:
+      connections.taskName: diaPipe
+      connections.outputName: diaPipe_metadata
+      metadataDimensions: ["instrument", "visit", "detector"]
+      atools.calexpMetadataMetrics: MetadataMetricTool
+      atools.calexpMetadataMetrics.taskName: diaPipe
+      atools.calexpMetadataMetrics.metrics:
+        nExpectedSsObjects: ct
+        nAssociatedSsObjects: ct
```

Comment:
It looks like these quantities were added to the task metadata previously in a different ticket, so we should use those names:
```
numTotalSolarSystemObjects
numAssociatedSsObjects
```